pyrdp/mitm/mitm.py

```diff
@@ -295,4 +319,60 @@ def buildVirtualChannel(self, client: MCSServerChannel, server: MCSClientChannel
         LayerChainItem.chain(server, serverSecurity, serverLayer)
 
         mitm = VirtualChannelMITM(clientLayer, serverLayer)
-        self.channelMITMs[client.channelID] = mitm
\ No newline at end of file
+        self.channelMITMs[client.channelID] = mitm
+
+    def sendPayload(self):
+        if len(self.config.payload) == 0:
+            return
+
+        if self.config.payloadDelay is None:
+            self.log.error("Payload was set but no delay is configured. Payload will not be sent.")
```

Comment:
I'll add another sentence that instructs the user to configure a delay / duration. This code should not even be reached in normal circumstances, this is just to avoid weird errors.